library/network_connections.py

```diff
@@ -2550,6 +2552,8 @@ def forget_nm_connection(self, path):
                 ]
             )
         except Exception:
+            # Assume that NetworkManager is not present.
```

Comment:
This is duplicating the doc string for the function, so IMHO it would be better to remove this and/or to properly ignore only the error cases when busctl is not available or if the NetworkManager dbus is not available instead of ignoring all exceptions. I feel  guilty since this is technical debt from me. :see_no_evil:

Reply:
@liangwen12year, @thom311, could you please take a look at this exception? Thanks!